packages/commons/src/fhir/task.ts

```diff
@@ -111,10 +114,14 @@ export type Task = Omit<
   | 'intent'
   | 'identifier'
   | 'code'
+  | 'focus'
 > & {
   lastModified: string
   status: 'ready' | 'requested' | 'draft' | 'accepted' | 'rejected'
   extension: Array<Extension>
+  focus: {
```

Comment:
Just a random type improvement for Task as it came up as part of the task. Don't think there is ever a case where a Task wouldn't reference a composition in `focus.reference`